ScoutSuite/output/data/inc-scoutsuite/scoutsuite.js

```diff
@@ -374,6 +382,35 @@ function toggleDetails(keyword, item) {
     $(id).toggle();
 };
 
+/**
+ * Toggles between light and dark themes
+ */
+function toggle_theme() {
+    if (document.getElementById("dark_theme").checked) {
+        this.set_theme(DARK_THEME)
+    }
+    else {
+        this.set_theme(LIGHT_THEME)
+    }
+};
+
+/**
+ * Sets the css file location received as the theme
+ * @param file
+ */
+function set_theme(file)
+{
+    var oldlink = document.getElementsByTagName("link").item(THEME_INDEX);
```

Comment:
I think adding a tag to the link element holding the theme would be more reliable. This solution breaks if the lines are moved around or if we add another link tag before.

Reply:
I think a light & dark themes are sufficient, but it would be nice if the implementation allows to easily swap themes.

Reply:
Now using an index, thanks for the tip @Aboisier ! @j4v Currently the only work needed to swap out themes is swapping inc-bootstrap/css/bootstrap-dark.min.css for a different file of the right version (3 atm) and changing the value of the const DARK_THEME in scoutsuite.js accordingly.